src/lesson7.java

```diff
@@ -0,0 +1,19 @@
+public class lesson7 {
+
+    public static void main(String [] args){
+        Cat cat1 = new Cat("Murzik",true ,25);
+        Cat cat2 = new Cat("Tom", true, 15);
+        Cat cat3 = new Cat("Cezar",true, 50);
+        Plates plates = new Plates(50);
+        Cat [] kitties = new Cat [3];
+        kitties[0] = cat1;
+        kitties[1]= cat2;
+        kitties[2] = cat3;
+
+       for (int count1=0; count1 < kitties.length; count1++) {
+           plates.FullfilmentPlate();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.